zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -23,14 +26,25 @@ use zebra_chain::transaction::MEMPOOL_TRANSACTION_COST_THRESHOLD;
 /// This also caches the all the spent outputs from the transactions in the mempool. The spent
 /// outputs include:
 ///
+/// - the dependencies of transactions that spent the outputs of other transactions in the mempool
+/// - the UTXOs of transactions in the mempool
```

Comment:
Shouldn't this be outputs as well?


```suggestion
/// - the outputs of transactions in the mempool
```